### ISSUE TITLE
[Standard Edition] Compose window: repeating dropdown email suggestions

### ISSUE THREAD

**Issue by megadr01d** [RainLoop/rainloop-webmail#821]
Consider the following contact example in the Address book:

```
First: John
Last: Knee
Nick: Johnny
Email 1
Email 2
Email 3
```

When entering `john` in the **To** field, we get 6 results instead of 3:

```
"John Knee" <Email 1> 
"Johnny" <Email 1>
"John Knee" <Email 2> 
"Johnny" <Email 2>
"John Knee" <Email 3> 
"Johnny" <Email 3>
```

First, Last and Nick should be grouped unless it's a limitation of the SQLite implementation.

**Comment by RainLoop** [comment]
Hi @megadr01d 
Do you use ownCloud package?

**Comment by megadr01d** [comment]
@RainLoop No, standard version (actually I was gonna specify that in the issue title but at the last minute I thought it was not relevant).

**Comment by RainLoop** [comment]
Please, try the latest version (1.9.3.363)

**Comment by megadr01d** [comment]
I'm afraid it's still not fixed in 1.9.3.363, tested in Safari (OS X) and Firefox (Windows).

![dropdown-email-suggestions](https://cloud.githubusercontent.com/assets/9649385/9976787/c61358ba-5ee6-11e5-8c6f-d421013110f8.jpg)

**Comment by RainLoop** [comment]
Hm, i don't see anything wrong in your screenshot.

**Comment by megadr01d** [comment]
The entry in the contacts is:

```
First: John
Last: Doe
Nick: Johnny
Email: john@mail.com
```

In the screenshot, the 2 entries are referring to the same entry. If John had 3 emails I would get 6 results when only 3 would be needed. Although Rainloop is filtering the word `john` correctly, my point is that, to reduce email repetition, the result in the screenshot above should show up as something like

```
"John Doe (Johnny)" <john@mail.com>
```

or in case he had 3 emails

```
"John Doe (Johnny)" <john@mail1.com>
"John Doe (Johnny)" <john@mail2.com>
"John Doe (Johnny)" <john@mail3.com>
```

**Comment by megadr01d** [comment]
This seems fixed in the latest 1.9.3.365. Only unique emails are shown for both first name, last name and nickname. :+1:
